src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String[] STATISTIC = {"supply", "buy"};
+    private static final String SEPARATOR = ",";
+    private static final String RESULT = "result";
+    private static final int AMOUNT_SUPPLY_INDEX = 0;
+    private static final int AMOUNT_BUY_INDEX = 1;
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readeFromFile(fromFileName);
+        String result = createResultReport(dataFromFile);
+        writeToFile(toFileName, result);
+    }
+
+    private String readeFromFile(String filename) {
+        File file = new File(filename);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String oneLine = reader.readLine();
+            while (oneLine != null) {
+                stringBuilder.append(oneLine).append(System.lineSeparator());
+                oneLine = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String createResultReport(String fileString) {
+        String[] lines = fileString.split(System.lineSeparator());
+        int[] amount = {0, 0};
```

Comment:
no need for this array, two local variables look better than relying on some custom format and accessing by indexes which necessitates two more constants